src/main/java/com/cburch/logisim/gui/menu/AssemblyWindow.java

```diff
@@ -57,17 +57,7 @@
 import java.util.prefs.Preferences;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import javax.swing.JButton;
-import javax.swing.JCheckBoxMenuItem;
-import javax.swing.JComboBox;
-import javax.swing.JEditorPane;
-import javax.swing.JFileChooser;
-import javax.swing.JLabel;
-import javax.swing.JMenu;
-import javax.swing.JMenuBar;
-import javax.swing.JMenuItem;
-import javax.swing.JPanel;
-import javax.swing.JScrollPane;
+import javax.swing.*;
```

Comment:
Personally, I prefer to explicitly import only the used classes.

Reply:
Fixed

Reply:
Thanks!